packages/@roots/bud-support/src/util/InterpolateHtmlPlugin.ts

```diff
@@ -16,8 +15,9 @@ export default class InterpolateHtmlPlugin
 
   /**
    * @property htmlWebpackPlugin
+   * @todo fix this type definition @webpack5
    */
-  public htmlWebpackPlugin: HtmlWebpackPlugin
+  public htmlWebpackPlugin: any
```

Comment:
Something weird happening with this type definition. Setting to `any` temporarily.